openapi/webhook/schema/turvo.yaml

```
@@ -0,0 +1,2442 @@
+        - SHIPMENT_ADD
+        - SHIPMENT_UPDATE
+        - SHIPMENT_STATUS_UPDATE
+
+    TurvoShipmentEventPayload:
+      title: TurvoShipmentEvent
+      description: Event `SHIPMENT_ADD` and `SHIPMENT_UPDATE`
+      type: object
+      required:
+        - eventName
+      properties:
+        eventName: 
```

yamllint

[error] trailing spaces

(trailing-spaces)